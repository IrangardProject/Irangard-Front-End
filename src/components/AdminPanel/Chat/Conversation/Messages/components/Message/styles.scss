@import '../../../../scss/variables/colors';
@import '../../../../scss/common';

.rcw-message {
  margin: 10px;
  display: flex;
  white-space: pre-wrap;
  word-wrap: break-word;

  &-client {
    flex-direction: row-reverse;
  }
}

.rcw-timestamp {
  font-size: 10px;
  margin-top: 5px;
}

.rcw-client {
  display: flex;
  flex-direction: column;
  margin-left: auto;

  .rcw-message-text {
    @include message-bubble($turqois-2);

    white-space: pre-wrap;
    word-wrap: break-word;
  }

  .rcw-timestamp {
    align-self: flex-end;
  }
}

.rcw-response {
  display: flex;
  flex-direction: column;
  align-items: flex-start;

  .rcw-message-text {
    @include message-bubble($grey-2);
  }
}

/* For markdown elements created with default styles */
.rcw-message-text {
  p {
    margin: 0;
  }

  img {
    width: 100%;
    object-fit: contain;
  }
}

.rcw-avatar {
  width: 40px;
  height: 40px;
  border-radius: 100%;
  margin-right: 10px;

  &-client {
    margin: 0 0 0 10px;
  }
}



.rcw-message-cont{

  border-radius: 0.25rem;
  display: flex;
  flex-direction: column;
  padding: 12px !important;
  border-radius: 1.15rem;
  position: relative;
  word-wrap: break-word;
  background-color: #e7f1fc; 
  margin: 38px 2px !important;
  box-shadow:  -4px 5px 10px -5px rgba(0, 0, 0, 0.65) !important;
  z-index: 2;
}


.rcw-message-cont:before {
  top: -0.1rem;
  content: "";
  height: 1rem;
  position: absolute;
  border-top-left-radius: 0.8rem 0.7rem;
  border-right: 1rem solid #e7f1fc;
  right: -0.35rem;
  transform: translate(0, -0.1rem);
  box-shadow: 0px 5px 5px -5px #e7f1fc !important;
  z-index: 1;
}

.rcw-message-cont:after{
  
  top: -0.1rem;
  content: "";
  height: 1rem;
  position: absolute;
  background-color: #fff;
  border-top-left-radius: 0.5rem;
  right: -40px;
  transform:translate(-30px, -2px);
  width: 10px;
  z-index: 1;
  
}





.rcw-message-cont-server{

  border-radius: 0.25rem;
  display: flex;
  flex-direction: column;
  padding: 12px !important;
  border-radius: 1.15rem;
  position: relative;
  color: #fff;
  word-wrap: break-word;
  background-color: #2d4c69; 
  margin: 35px 16px !important;
  box-shadow: 4px 5px 10px -5px rgba(0, 0, 0, 0.65) !important;
  z-index: 2;
}


.rcw-message-cont-server:before {
  top: 0rem;
  content: "";
  height: 1rem;
  position: absolute;
  border-top-right-radius: 0.8rem 0.7rem;
  border-left: 1rem solid #2d4c69;
  left: -0.30rem;
  transform: translate(0, -0.1rem);
  box-shadow: 0px 5px 5px -5px #2d4c69 !important;
}

.rcw-message-cont-server:after{
  
  top: -0.1rem;
  content: "";
  height: 1rem;
  position: absolute;
  background-color: #fff;
  border-top-right-radius: 0.6rem;
  left: 20px;
  transform: translate(-30px, 0px);
  width: 10px;
  
}