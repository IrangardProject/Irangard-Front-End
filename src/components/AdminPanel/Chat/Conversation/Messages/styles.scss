@import '../../scss/variables/colors';
@import '../../scss/common';

.rcw-messages-container-admin {
  background-color: $white;
  height: 75vh !important;
  overflow-y: scroll;
  padding-top: 10px;
  padding-left: 17px;
  -webkit-overflow-scrolling: touch;
}

.rcw-full-screen {
  .rcw-messages-container-admin {
    @include messages-container-fs;
  }
}

@media screen and (max-width: 800px) {
  .rcw-messages-container-admin {
    @include messages-container-fs;
  }
}

.rcw-message {
  width: 330px !important;
  word-wrap: break-word !important;
  text-align: right;
  justify-content: right !important;
  margin: 0px 0 -30px 0;
}
