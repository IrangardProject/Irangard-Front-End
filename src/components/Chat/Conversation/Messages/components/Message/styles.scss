@import '../../../../scss/variables/colors';
@import '../../../../scss/common';

.rcw-message {
  margin: 10px;
  display: flex;
  white-space: pre-wrap;
  word-wrap: break-word;

  &-client {
    flex-direction: row-reverse;
  }
}

.rcw-timestamp {
  font-size: 10px;
  margin-top: 5px;
}

.rcw-client {
  display: flex;
  flex-direction: column;
  margin-left: auto;

  .rcw-message-text {
    @include message-bubble($turqois-2);

    white-space: pre-wrap;
    word-wrap: break-word;
  }

  .rcw-timestamp {
    align-self: flex-end;
  }
}

.rcw-response {
  display: flex;
  flex-direction: column;
  align-items: flex-start;

  .rcw-message-text {
    @include message-bubble($grey-2);
  }
}

/* For markdown elements created with default styles */
.rcw-message-text {
  p {
    margin: 0;
  }

  img {
    width: 100%;
    object-fit: contain;
  }
}

.rcw-avatar {
  width: 40px;
  height: 40px;
  border-radius: 100%;
  margin-right: 10px;

  &-client {
    margin: 0 0 0 10px;
  }
}



.rcw-message-cont{

  border-radius: 0.25rem;
  display: flex;
  flex-direction: column;
  padding: 12px !important;
  border-radius: 1.15rem;
  position: relative;
  word-wrap: break-word;
  background-color: #e7f1fc; 
  margin: 30px -5px !important;
  box-shadow:  -4px 5px 10px -5px rgba(0, 0, 0, 0.65) !important;
  z-index: 2;
}


.rcw-message-cont:before {
  top: -0.1rem;
  content: "";
  height: 1rem;
  position: absolute;
  border-top-left-radius: 0.8rem 0.7rem;
  border-right: 1rem solid #e7f1fc;
  right: -0.35rem;
  transform: translate(0, -0.1rem);
  box-shadow: 0px 5px 5px -5px #e7f1fc !important;
  z-index: 1;
}

.rcw-message-cont:after{
  
  top: -0.1rem;
  content: "";
  height: 1rem;
  position: absolute;
  background-color: #fff;
  border-top-left-radius: 0.5rem;
  right: -40px;
  transform:translate(-30px, -2px);
  width: 10px;
  z-index: 1;
  
}