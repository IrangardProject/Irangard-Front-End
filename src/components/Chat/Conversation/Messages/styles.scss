@import '../../scss/variables/colors';
@import '../../scss/common';

.rcw-messages-container {
  background-color: $white;
  height: 50vh;
  max-height: 410px;
  overflow-y: scroll;
  padding-top: 10px;
  padding-left: 17px;
  -webkit-overflow-scrolling: touch;
}

.rcw-full-screen {

  .rcw-messages-container {
    @include messages-container-fs;
  }
}

@media screen and (max-width: 800px)  {
  .rcw-messages-container {
    @include messages-container-fs;
  }
}


.rcw-message{
  width: 330px !important;
  word-wrap: break-word !important;
  text-align: right;
  justify-content:right !important;
  margin: 0px 0 -30px 0;

}


.rcw-message-server{
  direction: ltr;
  justify-content: left !important;
}
