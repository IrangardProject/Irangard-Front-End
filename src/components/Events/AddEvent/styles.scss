@import '/src/theme/main.scss';

.add-event {
  margin: 30px 0;
  border: 2px solid black;
  border-radius: 8px;
  text-align: right;
  background: white;

  &__no-auth {
    background-color: $c-white;
    border-radius: 8px;
    margin: 32px 0;

    padding: 32px;
  }

  &__section {
    padding: 30px;
    margin-right: auto;
    margin-left: auto;
    justify-content: center;
    max-width: 1000px;
  }
  .guide {
    color: $c-gray-40;

    &.box {
      border: 2px solid $c-gray-20;
      border-radius: 8px;
      width: 350px;
      max-width: 100%;
      height: 350px;
      padding: 10px;
      align-items: center;
      text-align: center;
      display: flex;
      justify-content: center;
      align-items: center;
    }
  }
  .baseinfo-section {
    .basic-field {
      margin: 15px 0;
    }
    .title {
      margin-top: 20px;
    }
    .description {
      margin-top: 40px;
      color: rgb(27, 27, 27);
    }
    .event-tags-input {
      display: flex;
      flex-direction: row;
      .tags-input {
        max-width: 600px;
        width: 100%;
        margin-top: 10px;
        margin-bottom: 10px;
      }
      .tags-input-button {
        // padding-bottom: 50px;
        .add-tag-button {
          margin-top: 10px;
          background-color: #04aa6c;
          // margin-bottom: 10px;
          margin-left: 10px;
          max-width: 90px;
          max-height: 50px;
          font-size: 14px;   
        }
      }
    }
    .event-tags-entered {
      display: flex;
      flex-direction: row;
      .tag-chips {
        background-color: #F1F1F1;
        color: black;
        margin-top: 10px;
        margin-bottom: 10px;
        margin-left: 10px;
        width: 150px;
        .MuiChip-deleteIcon {
          color: #73777B;
        } 
      }
    }
    .question {
      margin-top: 50px;
      margin-bottom: 10px;
      .options {
        display: flex;
        justify-content: flex-end;
        margin-top: 10px;
        &__chip {
          padding: 8px 12px;
        }
        @media (max-width: 600px) {
          flex-direction: column;
          align-items: center;
          &__chip {
            width: 180px;
            display: flex;
            justify-content: space-between;
            text-align: center;
            
            span {
              text-align: center;
              width: 100%;
            }
          }
        }
      }
      .active-chip {
        .MuiChip-root {
          box-shadow: 0 0 0 1.2px inset #34e0a1;
          border-color: #04aa6c;
          background-color: white;
        }
      }
      span {
        .MuiChip-root {
          background-color: whitesmoke;
        }

        margin: 5px;
      }
    }
  }
  .map-section {
    .state-city-address {
      margin-top: 20px;
      padding-left: 20px;
      .address {
        margin-top: 20px;
      }
      .coordinates__title {
        margin-top: 40px;
      }
      .coordinates {
        margin-top: 15px;
      }
    }
    .location-info {
      display: flex;
      flex-direction: row;
      &__map {
        margin-right: auto;
        margin-left: 10px;
        z-index: 0;
      }
      @media (max-width: 900px) {
        flex-direction: column;
        &__map {
          margin-top: 30px;
          margin-right: 0;
          margin-left: 0;
        }
      }
    }
    .title {
      font-size: 21px;
      color: #080340;
      margin-bottom: 15px;
      margin-top: 20px;
    }
    .coordinates__title {
      margin-top: 30px;
    }
  }

  .hotel-rooms {
    &__groupfield {
      margin-bottom: 5px;
      display: flex;
    }

    &__field {
      margin: 5px;
    }
    .room-price {
      position: relative;
      display: inline-block;

      &::after {
        content: 'تومان';
        position: absolute;
        top: 50%;
        left: 10px;
        transform: translateY(-50%);
      }
    }
    .delete-btn {
      color: $c-golbehi2;
      background-color: $c-white;
      transition: all 0.3s;
      &:hover {
        background-color: $c-golbehi2;
        color: $c-white;
      }
    }

    .add-btn {
      color: $c-green2;
      background-color: $c-white;
      transition: all 0.3s;
      &:hover {
        background-color: $c-green2;
        color: $c-white;
      }
    }
    .btn-box {
      display: flex;
      align-items: center;
      gap: 10px;
    }
  }
  .field-label {
    padding-bottom: 3px;
  }
  .basic-field {
    max-width: 700px;
    .field-input {
      outline: 0;
      padding: 14px;
      border-radius: 4px;
      transition: border-color 0.2s;
      width: 100%;
      border: 1px solid rgb(173, 173, 173);

      &:focus {
        border-color: #04aa6c;
        border-width: 2px;
      }
      &.invalid {
        border-color: $c-golbehi2;
      }
    }
  }
}
.work-time,
.state-city,
.coordinates {
  display: flex;
  gap: 20px;
  &__field {
    margin: 10px 0;
  }
}

.state-city,
.coordinates {
  &__field {
    flex-grow: 1;
  }
}
.contact-section {
  .basic-field {
    margin: 15px 0;
    &.worktime {
      margin-top: 40px;
    }
  }
}
.moreinfo-section {
  .tags,
  .hotel-features {
    margin: 30px 0;
  }

  .hotel-features {
    .MuiSelect-select {
      min-width: 275px;
    }
  }
  .upload-images {
    .title {
      padding-top: 10px;
    }
    margin-top: 10px;
  }
}

.time-and-date-section {
  .title {
    font-size: 15px;
  }
  .description {
    margin-top: 10px;
    margin-bottom: 10px;
  }
  .date-inputs {
    display: flex;
    flex-direction: row;
    gap: 20px;

    .start-date-input {
      min-width: 100px;
      .date-input {
        display: none;
      }
    }
    .end-date-input {
      min-width: 100px;      
      .date-input {
        display: none;
      }
    }
  }
  .time-inputs {
    margin-top: 20px;
    margin-bottom: 20px;
    display: flex;
    flex-direction: row;
    gap: 20px;
    .start-time-input {
      min-width: 100px;
      width: 27%;
      display: flex;
      flex-direction: column;
      .field-input {
        margin-top: 6px;
        outline: 0;
        padding: 14px;
        border-radius: 4px;
        transition: border-color 0.2s;
        width: 100%;
        border: 1px solid rgb(173, 173, 173);
  
        &:focus {
          border-color: #04aa6c;
          border-width: 2px;
        }
        &.invalid {
          border-color: $c-golbehi2;
        }
      }
    }
    .end-time-input {
      min-width: 100px;
      width: 27%;
      display: flex;
      flex-direction: column;
      .field-input {
        margin-top: 6px;
        outline: 0;
        padding: 14px;
        border-radius: 4px;
        transition: border-color 0.2s;
        width: 100%;
        border: 1px solid rgb(173, 173, 173);
  
        &:focus {
          border-color: #04aa6c;
          border-width: 2px;
        }
        &.invalid {
          border-color: $c-golbehi2;
        }
      }
    }
  }
}

.additional-info-section {
  .title {
    font-size: 15px;
  }
  .is-free {
    margin-top: 20px;
  }
  .contact-section {
    margin-top: 20px;
    .title {
      font-size: 18px;
    }
  }
  .event-summary {
    margin-top: 20px;
    .summary-description-label {
      margin-top: 10px;
    }
  }
  .upload-images {
    margin-top: 20px;
    .title {
      padding-top: 10px;
      font-size: 18px;
    }
    .img-description-label {
      margin-top: 10px;
      margin-bottom: 10px;
    }
  }
}
