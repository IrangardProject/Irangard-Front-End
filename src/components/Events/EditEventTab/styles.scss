@import 'src/theme/main.scss';

.event-dashboard-edit {
  background-color: $c-white;
  border-radius: 8px;

  &__title {
    font-size: 24px;
  }

  &__choose-img-btn {
    margin: 16px auto;
    display: block;
    max-width: 240px;
  }

  &__img {
    margin: 16px 0;
    width: 100%;
    max-width: 480px;
    max-height: 480px;
    margin: auto;
    display: block;
    border-radius: 8px;
  }

  .date-input {
    display: none;
  }
}
