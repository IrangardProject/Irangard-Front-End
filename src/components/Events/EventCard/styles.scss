@import 'src/theme/main.scss';

// .event-card {
//   border: 1px solid #eee;
//   background-color: $c-white;
//   border-radius: 10px;
//   box-shadow: 2px 4px 10px rgba(0, 0, 0, 0.1);
//   padding: 16px;
//   margin-bottom: 10px;
//   display: flex;
//   align-items: center;

//   &:hover {
//     box-shadow: 0 0 30px $c-black-40;
//   }

//   &__img {
//     height: 160px;
//     width: 240px;
//     max-width: 100%;
//     display: block;
//     border-radius: 8px;
//     margin-left: 16px;
//   }

//   &__title {
//     font-size: 18px;
//     font-weight: 700;
//     margin-bottom: 12px;
//   }

//   &__cost-capacity {
//     display: flex;
//     align-items: center;
//   }

//   &__capacity {
//     color: #666;
//     font-size: 16px;
//     font-weight: 800;
//     margin-left: 16px;
//   }

//   &__cost {
//     color: #2979fe;
//     font-weight: 800;
//     font-size: 18px;
//   }

//   &__date {
//     display: flex;
//     flex-wrap: wrap;
//     align-items: center;
//   }

//   &__start,
//   &__end {
//     font-size: 14px;
//     margin: 8px 0;
//     font-weight: 700;
//     padding: 8px;
//     border-radius: 8px;
//     width: auto;
//   }

//   &__start {
//     color: #006a33;
//     background-color: #f7fafb;
//     border: 1px solid #006a33;
//     margin-left: 16px;
//   }

//   &__end {
//     color: #e40300;
//     background-color: #fdeded;
//     border: 1px solid #e40300;
//   }

//   &__more {
//     // background-color: #feb714;
//     // color: $c-black;
//     font-size: 14px;
//     font-weight: 700;
//     padding: 8px;
//     margin: 8px 0;
//     border: none;
//     width: 120px;
//     height: 40px;
//     background-color: #004f32;
//     border-radius: 32px;
//     -webkit-transition: background-color 0.2s linear;
//     -o-transition: background-color 0.2s linear;
//     transition: background-color 0.2s linear;
//     // padding: 5px 0px;

//     cursor: pointer;
//     @media (min-width: 800px) {
//       // margin: 0px 30px;
//       padding: 10px 0px;
//     }
//     @media (max-width:800px) {
//       width: auto;
//     }
//     &:hover {
//       background-color: #00aa6c;
//     }
//     p {
//       margin: 0;
//       vertical-align: middle;
//       font-size: 16px;
//       font-weight: bold;
//       color: #ffffff;
//       display: inline-block;
//     }
//   }
// }

// @media (max-width: 768px) {
//   .event-card {
//     flex-direction: column;

//     &__left {
//       display: flex;
//       flex-direction: column;
//       text-align: center;
//       width: 100%;
//     }

//     &__title {
//       margin: 8px 0;
//     }

//     &__date-and-time {
//       display: flex;
//       flex-direction: column;
//       align-items: center;
//     }
//   }
// }


.event-card {
  background-color: #ffffff;
  border-radius: 8px;
  display: flex;
  flex-direction: column;
  overflow: hidden;

  height: 400px;
  width: 320px;
  // padding: 10px;
  // height: 340px;
  // width: 270px;

  margin: 0 8px;
  &:hover {
    box-shadow: 0 0 30px $c-black-40;
  }

  &__img {
    width: 100%;
    min-height: 190px;
    height: 70%;
    border-radius: 8px;
    @media (max-width: 768px) {
      min-height: 80px;
    }

  }
  &__title {
    font-size: 16px;
    font-weight: 400;
    line-height: 1.5;
    padding: 0 16px;
    margin: 10px 0 5px;
    @media (max-width: 768px) {
      width: 80px;
      font-size: 10.5px;
      margin: 8px 10px 4px 4px;
    }
  }
  &__date {
    display: flex;
    align-items: center;
    margin-bottom: 15px;
    &__date-icon {
      font-size: 20px;
      margin-right: 12px;
      
      margin-left: 4px;
    }
    &__text {
      padding-top: 10px;
      font-size: 14px;
    }
  }
}

@media (max-width: 1286px) {
  .event-card {
    height: 340px;
    width: 270px;
  }
}

@media (max-width: 768px) {
  .event-card {
    flex-direction: column;
    &__title {
      width: 100%;
      font-size: 14px;
    }
  }
}
