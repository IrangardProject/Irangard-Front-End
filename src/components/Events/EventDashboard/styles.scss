@import 'src/theme/main.scss';

.event-dashboard {
  background-color: $c-white;
  border-radius: 8px;
  margin: 32px 0;
  padding: 32px;

  &__tabs {
    display: flex;
    flex-wrap: wrap;
  }

  &__tab {
    padding: 12px 12px;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    cursor: pointer;
    border-bottom: 4px solid rgba(0, 0, 0, 0.1);

    &:hover {
      background-color: #f1f1f1;
    }

    &--active {
      border-bottom: 4px solid #333;
    }
  }

  &__tabs-content {
    margin-top: 24px;
  }
}
