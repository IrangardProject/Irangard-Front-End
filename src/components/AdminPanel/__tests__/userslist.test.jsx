import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import { ThemeProvider } from '@mui/material/styles';
import { QueryClient, QueryClientProvider } from 'react-query';
import Demo from '../UsersList';
import { createTheme } from '@mui/material/styles';



const theme = createTheme({
  palette: {
    background: {
      paper: 'red',
    },
  },
  typography: {
    h5: {
      
    },
  },
});


const queryClient = new QueryClient();

describe('Demo', () => {
  test('renders correctly', () => {
    const component = shallow(
      <QueryClientProvider client={queryClient}>
        <ThemeProvider theme={theme}>
          <Demo />
        </ThemeProvider>
      </QueryClientProvider>
    );
    expect(component.exists()).toBe(true);
  });

  test('matches the snapshot', () => {
    const component = create(
      <QueryClientProvider client={queryClient}>
        <ThemeProvider theme={theme}>
          <Demo />
        </ThemeProvider>
      </QueryClientProvider>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
