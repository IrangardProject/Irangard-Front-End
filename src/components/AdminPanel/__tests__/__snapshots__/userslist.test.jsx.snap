// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo matches the snapshot 1`] = `
<div
  className="user-controller"
>
  <div
    className="MuiBox-root css-1i055ha"
  >
    <div
      className="MuiTypography-root MuiTypography-h5 css-1o64xmw-MuiTypography-root"
    >
      اضافه کردن کاربر جدید
    </div>
    <div
      className="bordering MuiBox-root css-16zzb8e"
    >
      <div
        className="login-modal__TabContent"
      >
        <form
          className="login-modal-form"
          onSubmit={[Function]}
        >
          <div
            className="form__group field"
          >
            <input
              className="form__field"
              id="username"
              name="username"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="username"
              type="input"
            />
            <label
              className="form__label"
              htmlFor="username"
            >
              نام کاربری
            </label>
          </div>
          <div
            className="form__group field"
          >
            <input
              className="form__field"
              id="email"
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="email"
              type="input"
            />
            <label
              className="form__label"
              htmlFor="email"
            >
              ایمیل
            </label>
          </div>
          <div
            className="form__group field"
          >
            <input
              className="form__field"
              id="password"
              name="password"
              onBlur={[Function]}
              onChange={[Function]}
              placeholder="password"
              type="input"
            />
            <label
              className="form__label"
              htmlFor="password"
            >
              رمز عبور
            </label>
          </div>
          <input
            className="submit-btn"
            type="submit"
            value="ثبت"
          />
        </form>
      </div>
    </div>
  </div>
  <div
    className="MuiBox-root css-15jatt2"
  >
    <div
      className="MuiFormGroup-root MuiFormGroup-row css-qfz70r-MuiFormGroup-root"
    />
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12 css-13i4rnv-MuiGrid-root"
    >
      <div
        className="MuiTypography-root MuiTypography-h5 css-1o64xmw-MuiTypography-root"
      >
        لیست کاربران
      </div>
      <div
        className="css-1hos8tk"
      >
        <ul
          className="MuiList-root MuiList-padding css-h4y409-MuiList-root"
        />
      </div>
    </div>
  </div>
</div>
`;
